p_sets/2/encryption.rb

```diff
@@ -0,0 +1,43 @@
+module Encryption
+
+  ASCII_LETTERS = {
+    capitals: (65..90),
+    lowercase: (97..122)
+  }
+
+  def cyphertext_reporter(plaintext, integer)
+    puts format('%s', encrypt(plaintext, integer))
+  end
+
+  def encrypt(plaintext, string)
```

Comment:
I opted for string and integer because I have two make_key methods that produce workable keys from the user inputs. But from a user pov i see that their inputs are the keys, so i should rename the variables passed from make_key to the encryption methods maybe?